packages/chat-client/src/client.ts

```diff
@@ -237,16 +239,55 @@ export class ChatClient extends IChatClient {
     return this.events.removeListener(name, listener);
   };
 
+  public initSyncStores: IChatClient["initSyncStores"] = async ({
+    account,
+    signature,
+  }) => {
+    if (!this.syncClient) return;
+
+    this.chatSentInvites = new SyncStore(
+      CHAT_SENT_INVITES_CONTEXT,
+      this.syncClient,
+      account,
+      signature
+    );
+
+    this.chatThreads = new SyncStore(
+      CHAT_THREADS_CONTEXT,
+      this.syncClient,
+      account,
+      signature
+    );
+
+    await this.chatSentInvites.init();
+    await this.chatReceivedInvites.init();
+    await this.chatThreads.init();
+  };
+
   // ---------- Private ----------------------------------------------- //
 
   private async initialize() {
     this.logger.trace(`Initialized`);
     try {
+      this.syncClient = await SyncClient.init({
+        core: this.core,
+        logger: this.logger,
+      });
+
+      // Use active account to init stores
+      if (this.syncClient && this.syncClient.signatures.length > 0) {
+        const signatureEntry = this.syncClient.signatures.getAll({
+          active: true,
+        })[0];
+        await this.initSyncStores({
+          account: signatureEntry.account,
+          signature: signatureEntry.signature,
+        });
+      }
+
       await this.core.start();
-      await this.chatSentInvites.init();
-      await this.chatReceivedInvites.init();
-      await this.chatThreads.init();
       await this.chatMessages.init();
+      await this.chatReceivedInvites.init();
```

Comment:
Looks like `chatReceivedInvites` is now being init'd both here and in the `initSyncStores` helper